BMS Workspace-git/bms-1227-fw/Phantom_drivers/include/bms_data.h

```diff
@@ -9,23 +9,48 @@
 
 #include "sys_common.h"
 
+typedef enum State_t
+{
+    BMS_RUNNING,
+    BMS_CHARGING,
+    BMS_FAULT
+}State_t;
+
 typedef struct flags
 {
 	uint8_t BALANCE_EN; // If 1, then cell balancing is enabled (only done during charging), if 0, then cell balancing is disabled
 						// TODO: Set this = 1 if state machine evaluates to CHARGING state
 	uint8_t FUSE_FAULT; // If 1, then fuse has blown, if 0, then fuses are intact
+	uint8_t 3SECOND_FLAG; // If 1, then a cell has been in fault for longer than 3 seconds, else 0
+						  // TODO: If 1, put BMS in FAULT state
+	uint8_t TOTAL_CELL_ERROR_FLAG; // If 1, then more than four cells have been in fault during a check, else 0
+								   // TODO: If 1, put BMS in FAULT state
+	uint8_t BAD_SLAVE_CONNECTION_FLAG; // If 1, then BMS master is unable to read/write to/from at least one BMS slave
+									   // TODO: If 1, put BMS in FAULT state
 } flags;
 
-typedef struct SOCInfo
+typedef struct data
```

Comment:
Same as above comment

Reply:
Changed all member variable types to start with "bms" (ex: data -> bmsData)